src/components/Lookup/icons/closeIconDark.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function CloseIcon(props) {
+    const { className, style } = props;
+    return (
+        <svg
+            className={className}
+            style={style}
+            fill="#576574"
```

Comment:
as this is exact the same icon let's use only one icon (closeIcon) and pass a color prop with '#01b6f5' as default then when a value is selected use:
```<CloseIcon color="#576574" />